### PR TITLE
removed default box-shadow

### PR DESCRIPTION
**Pull request by shubhampandeyhld**
After clicking the buttons an default box-shadow was added which was not looking good.

Issue:

### CODE REVIEW

**Comment by shahednasser** [comment]
Thank you for contributing! The box-shadow for the buttons is important on click to show that the button was clicked (this is more for user experience). If you think the current box-shadow doesn't look good, then please provide an alternative that can show the user that they indeed clicked the button.

**Comment by shubhampandeyhld** [comment]
Okay I'll come with an alternative ,meanwhile I am also adding og tags for seo but their is no site logo present here.

**Comment by shahednasser** [comment]
Yes, we currently don't have one. There's an issue about it (#2 ) but nobody has contributed yet, unfortunately.  
If you're working on og tags then just put the tags in for now and once there's a logo we'll add it in.

**Comment by shahednasser** [comment]
@shubhampandeyhld  I saw you added the tags for SEO. Will you add something else regarding the box shadow? If not please remove the current changes to the box shadow so I can merge the changes for og tags!

**Comment by shubhampandeyhld** [comment]
okay..

File: assets/css/index.css [hunk 1]
```diff
@@ -121,6 +121,10 @@ section {
   margin-bottom: 1em;
   font-size: 1.5rem;
 }
+.button-container .sbtn :hover{
```
Comment (shahednasser) [inline assets/css/index.css:124]: Please remove this as well. I just need the changes in `index.html`

**Comment by shubhampandeyhld** [comment]
Hey I have just added the default box-shadow,  so now you can merge the og tags.

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: assets/css/index.css (hunk 1 above)
Comment (shubhampandeyhld) [reply assets/css/index.css:124]: yes , I am removing it

**Comment by shahednasser** [comment]
Great! Thank you!

@all-contributors please add @shubhampandeyhld for code and ideas

**Pull request merged**